runtime-data/aosc-os-presets-genericcloud/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=aosc-os-presets-genericcloud
+PKGSEC=misc
+PKGDEP=""
+PKGDES="Systemd presets for AOSC OS GenericCloud distributions"
```

Comment:
I would just call this package `aosc-os-presets-cloud` since there is no contrast between Azure and "generic cloud."

Reply:
Systemd presets for AOSC OS hosted on the cloud